lib/runtime/wasmer/exports_test.go

```diff
@@ -408,6 +408,32 @@ func TestInstance_GrandpaAuthorities_PolkadotRuntime(t *testing.T) {
 	require.Equal(t, expected, auths)
 }
 
+func TestInstance_BabeGenerateKeyOwnershipProof_NodeRuntime(t *testing.T) {
+	tt := trie.NewEmptyTrie()
+	rt := NewTestInstanceWithTrie(t, runtime.NODE_RUNTIME, tt)
+	authorityID := types.AuthorityID{}
+	slot := uint64(1)
+	_, err := rt.BabeGenerateKeyOwnershipProof(slot, authorityID)
+	require.NoError(t, err)
+}
+
+func TestInstance_BabeSubmitReportEquivocationUnsignedExtrinsic_NodeRuntime_(t *testing.T) {
+	tt := trie.NewEmptyTrie()
+	rt := NewTestInstanceWithTrie(t, runtime.NODE_RUNTIME, tt)
+	authorityID := types.AuthorityID{}
```

Comment:
you should test using non-empty values to ensure they are passed around correctly in the call stack

```suggestion
	authorityID := types.AuthorityID{1}
```